bin/mark_changepoints_in_json

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+"""
+Write changepoint and classification information into JSON files.
+"""
+
+import argparse
+import copy
+import os
+import os.path
+import rpy2
+import rpy2.interactive.packages
+import rpy2.robjects
+import sys
+
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+from warmup.krun_results import read_krun_results_file, write_krun_results_file
+
+
+def main(in_files, window_size):
+    cpt = rpy2.interactive.packages.importr('changepoint')  # Import once.
```

Comment:
We import the R package once, and pass it to the method where it is used, rather than importing it inside the method.